routesrv/eskipbytes.go

```diff
@@ -15,8 +17,10 @@ import (
 // serve as an HTTP handler exposing its content.
 type eskipBytes struct {
 	data        []byte
+	hash        string
 	initialized bool
 	mu          sync.RWMutex
+	useEtag     bool
```

Comment:
Do we need this to be configurable? Maybe we should always calculate etag and leave it up to the client to use it if they like.